lib/Push.php

```diff
@@ -246,6 +246,8 @@ public function pushToDevice(int $id, INotification $notification, ?OutputInterf
 			$language = $this->l10nFactory->getUserLanguage($user);
 			$this->printInfo('Language is set to ' . $language);
 
+			$this->notificationManager->preloadDataForParsing([$notification], $language);
```

Comment:
Tried moving Talk to `IPreloadableNotifier` I noticed this should be inside the `$this->notificationManager->setPreparingPushNotification(true);` wrapping, so apps can optimize the push case better.

In most such cases should be loaded/aware already anyway?

Reply:
Or should we even add a "preload reason" argument or something?

Because:
- when pushing we will prepare the same notification for a set of different users (N:1)
- when preparing emails we will face different users with different items (N:M)
- and on endpoint controller we are preparing for a single user but different subjects (1:M)

Depending on the case it can can make sense to load data from different directions?